content/docs/uncontrolled-components.md

```diff
@@ -37,15 +37,15 @@ class NameForm extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/WooRWa?editors=0010)
+[**Pruébalo en CodePen**](https://codepen.io/gaearon/pen/WooRWa?editors=0010)
 
-Since an uncontrolled component keeps the source of truth in the DOM, it is sometimes easier to integrate React and non-React code when using uncontrolled components. It can also be slightly less code if you want to be quick and dirty. Otherwise, you should usually use controlled components.
+Ya que un componente no controlado mantiene la fuente de la verdad en el DOM, a veces es más fácil integrar el código React y el código no React cuando usas componentes no controlados. También puede ser un poco menos de código si quieres ser rápido y sucio. De lo contrario, normalmente debes utilizar componentes controlados.
 
-If it's still not clear which type of component you should use for a particular situation, you might find [this article on controlled versus uncontrolled inputs](http://goshakkk.name/controlled-vs-uncontrolled-inputs-react/) to be helpful.
+Sí aún no tienes claro qué tipo de componente debes usar para una situación en particular, puedes encontrar [este articulo sobre entradas controladas y no controladas](http://goshakkk.name/controlled-vs-uncontrolled-inputs-react/) para ser tu ayuda.
```

Comment:
There is a typo in "Sí". Also, I would rephrase the last part to make it sound more idiomatic.
```suggestion
Si aún no tienes claro qué tipo de componente debes usar para una situación en particular, [este artículo sobre entradas controladas y no controladas](http://goshakkk.name/controlled-vs-uncontrolled-inputs-react/) te puede resultar útil.
```